bindings/matrix-sdk-crypto-ffi/src/device.rs

```diff
@@ -25,6 +25,9 @@ pub struct Device {
     /// Is our cross signing identity trusted and does the identity trust the
     /// device.
     pub cross_signing_trusted: bool,
+    /// The first time this device was seen in local timestamp, seconds since
+    /// epoch
```

Comment:
```suggestion
    /// The first time this device was seen in local timestamp, seconds since
    /// epoch.
```

Local is “server local”, “client local” or “device local”?

Reply:
client local, will update comment